src/pages/index.tsx

```diff
@@ -31,119 +32,107 @@ import {
 export const actions = [
   {
     title: 'What is Uniswap',
-    href: '#',
     icon: InformationCircleIcon,
-    to: './concepts/overview',
-    text: `Learn about the core concepts of the Uniswap Protocol. Swaps, Pools, Concentrated Liquidity and more.`,
-    name: ElementName.WHAT_IS_UNISWAP,
+    to: '/concepts/overview',
+    text: `Learn about the core concepts of the Uniswap Protocol, Swaps, Pools, Concentrated Liquidity and more.`,
   },
   {
     title: 'Integrate with Uniswap',
-    href: '#',
     icon: QuestionMarkCircleIcon,
-    to: './sdk/v3/overview',
+    to: '/sdk/v3/overview',
     text: `Learn how to integrate with Uniswap by building a dApp through guided examples.`,
-    name: ElementName.V3_SDK,
   },
   {
     title: 'The Uniswap smart contracts',
-    href: '#',
     icon: BookOpenIcon,
-    to: './contracts/v3/overview',
-    text: `Learn about the architecture of the Uniswap Protocol smart contracts through guided examples`,
-    name: ElementName.SMART_CONTRACT_OVERVIEW,
+    to: '/contracts/v3/overview',
+    text: `Learn about the architecture of the Uniswap Protocol smart contracts through guided examples.`,
   },
 ]
 
-export const github = [
+export const developerLinks = [
   {
     title: 'uniswap-v3-core',
     href: 'https://github.com/Uniswap/uniswap-v3-core',
     icon: CodeIcon,
-    name: ElementName.V3_CORE,
```

Comment:
Why are the element names removed here?

Reply:
These keeps changing (and was proposed earlier) and so it's prudent to id them by the item they link to.